.github/workflows/rust-pr-close.yml

```
@@ -0,0 +1,20 @@
+name: Rust PR Closed
+
+on:
+    pull_request:
+      branches: ["main"]
+      types: [closed]
+
+jobs:
+  delete-tag:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to Docker Hub
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)